packages/server-wallet/server-wallet.api.md

```diff
@@ -377,7 +377,9 @@ export class SingleThreadedEngine extends EventEmitter<EventEmitterType> impleme
     //
     // (undocumented)
     channelFinalized(arg: ChannelFinalizedArg): Promise<void>;
-    closeChannel({ channelId }: CloseChannelParams): Promise<SingleChannelOutput>;
+    closeChannel({ channelId, }: CloseChannelParams): Promise<SingleChannelOutput & {
+        newObjective: WalletObjective;
+    }>;
     closeChannels(channelIds: Bytes32[]): Promise<MultipleChannelOutput>;
```

Comment:
Have you thought about how the new `closeChannel` engine method should relate to the existing `closeChannels` engine method?

Reply:
The `CloseChannel` method isn't new but has been updated so the return type always returns includes a `newObjective`.  This is mostly for convenience so consumers of `createChannel` don't have to check if `newObjective` is defined or not.

I think it would be trickier to do the same with `CloseChannels` and since we're not using it I don't think it's worth the effort.

Longer term we may want to look at retiring some of the engine methods that are no longer used.